### DISCUSSION TITLE
Wiki page for tested hardware?

### DISCUSSION THREAD

**Discussion by Chewie9999** [PartialVolume/shredos.x86_64#114] [Ideas]
Does anyone think it would be worth having a wiki page showing hardware that has been known to work with ShredOS?
Eg, Make, Model, BIOS version, working/non-working shredOS version, boot method (CD, USB, virtual CD (Zalman, Ventoy etc)).

Personally I think it would be nice to keep track of what people have tried, and it might show up patterns of non-working hardware (like issue #111 )

**Comment by PartialVolume** [comment]
I think that would be useful however I would maybe want to take it a step further .... and some people might not be keen on this, but I've been thinking it would be really useful to automate that process. The user would have to enable the feature, i.e opt in, and have ShredOS contact shredos.com (the domain is currently owned by me but remains unused at present) and upload hardware details, i.e only hardware make/model data, no identifiable data obviously. The data would then be displayed in real time with maybe tabulated data and graphs. ShredOS would obviously have to be running on a internet connected network. The user could also manually add  information about systems that work or don't work to the database. Maybe even a grand total of the number of completed wipes ShredOS has carried out. As long as people opted in that would generate a significant amount of data on what hardware ShredOS was successfully or not, running on.

**Reply by nuneze54** [reply]
I don't want to take away your resources from other features which IMO are more important, but this sounds like a great feature to me! I would gladly test all "my" hardware again.

**Comment by PartialVolume** [comment]
@Chewie9999 Moved from issues to discussions [ideas]

**Reply by Chewie9999** [reply]
That's fine with me, although a manual way would be good too (or via a file saved on the USB boot device you can transfer later), as a lot of the time I use machines that are not connected to a network.
Whatever is easier for you. I don't want to make it a hassle for you, I was hoping it would help people to decide which hardware to use for wiping without causing you loads of support time answering questions about whether specific hardware was supported or not.
